tests/fixtures/valid_message.json

```diff
@@ -32,6 +33,16 @@
 			"name": "Bob",
 			"wallet": "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB"
 		},
+		"cc": [
+      {
+        "name": "Alice",
+        "wallet": "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
```

Comment:
Hm, this probably should have triggered a checksum error. I wonder why it didn't?

Reply:
I started down the path of enforcing checksum. To my great horror, the checksum version of the address did not encode the same way, it changed the hash!

So I settled for increasing the warning in the docs about the lack of tests.